addon/styles/addon.scss

```diff
@@ -4,64 +4,65 @@
   position: relative;
   display: flex;
   flex-direction: row;
+
   .file-select-overlay {
     position: absolute;
+    top: 3px;
+    left: 2px;
     width: 100%;
-    height: 100%;
+    height: 35px;
     display: flex;
     flex-direction: row;
     opacity: 0;
+
+    .text {
```

Comment:
I'd prefer that we start fixing this type of style to be `&-text` so that the final style is

.file-select-overlay-text

As a method of defensive scoping